test/test_specs.py

```diff
@@ -2146,36 +2151,14 @@ def test_stack_index_multdim(self, stack_dim):
             assert c[:, :, 0, ...] is c1
             assert c[:, :, 1, ...] is c2
 
-    @pytest.mark.parametrize("stack_dim", [0, 1, 2, -3, -2, -1])
-    def test_stack_expand_one(self, stack_dim):
-        c1 = CompositeSpec(a=UnboundedContinuousTensorSpec(shape=(1, 3)), shape=(1, 3))
-        c = torch.stack([c1], stack_dim)
-        if stack_dim in (0, -3):
-            c_expand = c.expand([4, 2, 1, 3])
-            assert c_expand.shape == torch.Size([4, 2, 1, 3])
-            assert c_expand.dim == 1
-        elif stack_dim in (1, -2):
-            c_expand = c.expand([4, 1, 2, 3])
-            assert c_expand.shape == torch.Size([4, 1, 2, 3])
-            assert c_expand.dim == 2
-        elif stack_dim in (2, -1):
-            c_expand = c.expand(
-                [
-                    4,
-                    1,
-                    3,
-                    2,
-                ]
-            )
-            assert c_expand.shape == torch.Size([4, 1, 3, 2])
-            assert c_expand.dim == 3
-        else:
-            raise NotImplementedError
-
```

Comment:
I couldn't see how this test could be made to make sense anymore. Even if you instantiate `c` directly as a `LazyStackedCompositeSpec`

```python
c = LazyStackedCompositeSpec(c1, dim=stack_dim)
```

then because `.expand` uses `torch.stack` internally, you still wind up with a regular `CompositeSpec` which breaks the checks.